file_formats/cmx_format.mli

```diff
@@ -46,7 +46,8 @@ type unit_infos =
     mutable ui_send_fun: int list;        (* Send functions needed *)
     mutable ui_export_info: export_info;
     mutable ui_force_link: bool;          (* Always linked *)
-    mutable ui_for_pack: string option }  (* Part of a pack *)
+    mutable ui_for_pack: string option;   (* Part of a pack *)
+    mutable ui_need_stdlib: bool}         (* caml_standard_library_nat needed *)
```

Comment:
What is the release process for `.cmx` changes? In the semver world this would be a backwards-incompatibility change, and that would require a major version bump. OCaml obviously doesn't do that. Instead, I think it implicitly treats each minor version change as possibly backwards-incompatible, and explicitly bumps Misc.Magic_number when necessary??

Do we need to bump Misc.Magic_number in this commit?

Reply:
The magic numbers are bumped unconditionally at each release, so that compiler artifacts are _never_ compatible across different major versions of the compiler.